### ISSUE TITLE
Should we remove carousel tile from the package core?

### ISSUE THREAD

**Issue by hvelarde** [collective/collective.cover#485]
Lately we have had some discussions about many things on the carousel tile and some pull requests have been proposed to offer an alternative solution

* Replacing Galleria with Cycle2 (#442)
* Displaying a collection in a Bootstrap carousel (#470)

There are many solutions for carousels and everybody seems to have their own opinion on which one is the best.

So… should we just spin off the code of the carousel tile out of the core of this package to make it easier to maintain and leave people choose whatever JS plug-in they want to use?

**Comment by djowett** [comment]
I think yes... but smoothing the migration could do with some bright ideas

**Comment by rodfersou** [comment]
I agree with the idea too

**Comment by fredvd** [comment]
Can't disagree here, suggested it in #442 :-)

**Comment by espenmn** [comment]
Yes. (and also, if it should be of any interest, I started work with  tiles for collective.plonetruegallery https://pypi.python.org/pypi/collective.ptg.tile/0.1b1 (worked when I made it, but has never been used on any sites)

**Comment by hvelarde** [comment]
@espenmn feel free to document that on the README file

**Comment by djowett** [comment]
Just to log (for those who don't already know) that https://github.com/collective/covertile.cycle2 is a new carousel tile based on the Cycle2 framework. It could do with some review & feedback from any interested party before it's first release

**Comment by hvelarde** [comment]
@djowett we are going to start using it tomorrow; we will let you know.

**Comment by djowett** [comment]
Correct me if I'm wrong, but this is not question any more...

**Comment by hvelarde** [comment]
no, I will open a branch later today to address this issue and will move the code of the Galleria carousel to the new package.
